Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendAllPANWIoTDevicesToCiscoISE/README.md

```diff
@@ -0,0 +1,37 @@
+This scripts retrieves all devices from PANW IoT cloud and exports them as Endpoints with custom attributes to Cisco ISE
```

Comment:
Retrieves all devices from PANW IoT Cloud and exports them as endpoints with custom attributes to Cisco ISE.